modules/common/time/time.h

```diff
@@ -147,6 +147,14 @@ class Clock {
 
 inline Clock::Clock() {
   mode_ = FLAGS_use_cyber_time ? ClockMode::CYBER : ClockMode::SYSTEM;
+  // Force to use Cyber time is USE_SIM_TIME is set
+  const char* use_sim_time = ::getenv("USE_SIM_TIME");
+  if (use_sim_time != nullptr) {
+    std::string use_sim_time_str(use_sim_time);
+    if (use_sim_time_str == "true") {
```

Comment:
```
export USE_SIM_TIME=1
export USE_SIM_TIME=true
```
The two forms listed above should both be OK.

Reply:
I'm against ENV-based behavior which is very hard to debug. Prefer FLAG.

Reply:
I did intend to use gflag, particularly since Apollo already has a flag named "use_cyber_time". But I did not figure out a way to set it to true (its default value is false as declared[ here](https://github.com/ApolloAuto/apollo/blob/master/modules/common/configs/config_gflags.cc#L46). Can you tell me how to change this gflag? I would like to dynamically set it via a command line argument, so that we can switch among different modes.

Reply:
Simply `--use_cyber_time` will set it to true.

Reply:
Using --use_cyber_time with what command? "bootstraph_lgsvl.sh --use_cyber_time", "cyber_launch start xxx/xxx.launch --use_cyber_time", or something else?

Reply:
We use
https://github.com/ApolloAuto/apollo/blob/master/modules/common/data/global_flagfile.txt
to config shared flags across modules, if their own flagfiles inherit from the global one correctly: https://github.com/ApolloAuto/apollo/blob/master/modules/control/conf/control.conf#L1

Reply:
But "use_cyber_time" is already set to "true" in global_flagfile. However, I still see it is false in the code. This is why I am asking hot to set it. This global_flagfile seems not working?